### ISSUE TITLE
Gluu Server 3.0.1 fails to start after VM reboot

### ISSUE THREAD

**Issue by zamilskhan** [GluuFederation/community-edition-setup#274]
OS: CentOS 6.8
Package: 3.0.1 2-1.centos6
VM was rebooted after a fresh installation.
```
[root@centos ~]# service gluu-server-3.0.1 start
ERROR: Can't start gluu server.
HINT: Please manually remove /var/run/gluu-server-3.0.1.pid and unmount chroot container by running
rm -f /var/run/gluu-server-3.0.1.pid
df -aP | grep gluu-server-3.0.1 | awk '{print $6}' | xargs -I {} umount -l {}
[root@centos ~]# rm -f /var/run/gluu-server-3.0.1.pid
[root@centos ~]# df -aP | grep gluu-server-3.0.1 | awk '{print $6}' | xargs -I {} umount -l {}
umount: /opt/gluu-server-3.0.1/dev/pts: not found
[root@centos ~]# service gluu-server-3.0.1 start                                                                                                      
ERROR: Can't start gluu server.
HINT: Please manually remove /var/run/gluu-server-3.0.1.pid and unmount chroot container by running
rm -f /var/run/gluu-server-3.0.1.pid
df -aP | grep gluu-server-3.0.1 | awk '{print $6}' | xargs -I {} umount -l {}
[root@centos ~]# service gluu-server-3.0.1 status
ERROR: gluu server was not run properly.
HINT: Please manually remove /var/run/gluu-server-3.0.1.pid and unmount chroot container by running
rm -f /var/run/gluu-server-3.0.1.pid
df -aP | grep gluu-server-3.0.1 | awk '{print $6}' | xargs -I {} umount -l {}
```

**Comment by alvesadrian** [comment]
a customer reported the same on debs, we gonna need to add update-rc.d gluu-server-3.0.1 defaults on setup py or in postinstall of the pkg and the same with chkconfig gluu-server-3.0.1 on rpms

**Comment by adrian-gluu** [comment]
a customer reported the same on debs, we gonna need to add update-rc.d gluu-server-3.0.1 defaults on setup py or in postinstall of the pkg and the same with chkconfig gluu-server-3.0.1 on rpms

**Comment by zamilskhan** [comment]
This issue is not present in CentOS 7.2.

**Comment by mzico** [comment]
@zamilskhan @adrian-gluu 

Two things to share... 

I faced same for Ubuntu 14.04 as well... 

Here are couple of commands added below. See after starting and logging into the system; just oxauth ( and solserver started ). idp,identity and apache was down. 
I had to manually start them and after that... check out those java processes: 

```
root@ubuntu:~# service gluu-server-3.0.1 start
root@ubuntu:~# service gluu-server-3.0.1 login
gluu-server-3.0.1 is running...
logging in...
root@ce301:~# ps ax | grep java
 30843 ?        Sl     0:15 /opt/jre/bin/java -server -Xms256m -Xmx1084m -XX:+DisableExplicitGC -Dgluu.base=/etc/gluu -Dpython.home=/opt/jython -Dcatalina.base=/opt/gluu/jetty/oxauth -Djetty.logging.dir=/opt/gluu/jetty/oxauth/logs -Djetty.home=/opt/jetty -Djetty.base=/opt/gluu/jetty/oxauth -Djava.io.tmpdir=/opt/jetty-9.3/temp -jar /opt/jetty/start.jar jetty.http.host=localhost jetty.http.port=8081 jetty.state=/opt/gluu/jetty/oxauth/oxauth.state jetty-logging.xml jetty-started.xml start-log-file=/opt/gluu/jetty/oxauth/logs/start.log
 31199 pts/8    S+     0:00 grep --color=auto java
root@ce301:~# service identity start
Starting Jetty: . . . . . . . OK Fri Mar  3 15:12:24 UTC 2017
root@ce301:~# service idp start
Starting Jetty: . . . . OK Fri Mar  3 15:12:49 UTC 2017
root@ce301:~# service apache2 start
 * Starting Apache httpd web server apache2                                                                                                                                                             *
root@ce301:~# ps ax | grep java
 30843 ?        Sl     0:15 /opt/jre/bin/java -server -Xms256m -Xmx1084m -XX:+DisableExplicitGC -Dgluu.base=/etc/gluu -Dpython.home=/opt/jython -Dcatalina.base=/opt/gluu/jetty/oxauth -Djetty.logging.dir=/opt/gluu/jetty/oxauth/logs -Djetty.home=/opt/jetty -Djetty.base=/opt/gluu/jetty/oxauth -Djava.io.tmpdir=/opt/jetty-9.3/temp -jar /opt/jetty/start.jar jetty.http.host=localhost jetty.http.port=8081 jetty.state=/opt/gluu/jetty/oxauth/oxauth.state jetty-logging.xml jetty-started.xml start-log-file=/opt/gluu/jetty/oxauth/logs/start.log
 31228 ?        Sl     0:29 /opt/jre/bin/java -server -Xms256m -Xmx723m -XX:+DisableExplicitGC -Dgluu.base=/etc/gluu -Dcatalina.base=/opt/gluu/jetty/identity -Dpython.home=/opt/jython -Dorg.eclipse.jetty.server.Request.maxFormContentSize=50000000 -Djetty.logging.dir=/opt/gluu/jetty/identity/logs -Djetty.home=/opt/jetty -Djetty.base=/opt/gluu/jetty/identity -Djava.io.tmpdir=/opt/jetty-9.3/temp -jar /opt/jetty/start.jar jetty.http.host=localhost jetty.http.port=8082 jetty.state=/opt/gluu/jetty/identity/identity.state jetty-logging.xml jetty-started.xml start-log-file=/opt/gluu/jetty/identity/logs/start.log
 31365 ?        Sl     0:18 /opt/jre/bin/java -server -Xms256m -Xmx723m -XX:+DisableExplicitGC -Dgluu.base=/etc/gluu -Dcatalina.base=/opt/gluu/jetty/idp -Djetty.logging.dir=/opt/gluu/jetty/idp/logs -Djetty.home=/opt/jetty -Djetty.base=/opt/gluu/jetty/idp -Djava.io.tmpdir=/opt/jetty-9.3/temp -jar /opt/jetty/start.jar jetty.http.host=localhost jetty.http.port=8086 jetty.state=/opt/gluu/jetty/idp/idp.state jetty-logging.xml jetty-started.xml start-log-file=/opt/gluu/jetty/idp/logs/start.log
 31462 pts/8    S+     0:00 grep --color=auto java
```

**Comment by yurem** [comment]
@adrian-gluu is going to add to post-install script commands to configure automatic service restart after VM reboot.

**Comment by adrian-gluu** [comment]
i just did that for ubuntu14/16 debian8 rhel6 and centos6, next build will fix this

**Comment by zamilskhan** [comment]
Fixed in `gluu-server-3.0.1-1-1`
